apps/server/src/shared/domain/service/task-copy.service.ts

```diff
@@ -25,22 +25,7 @@ export class TaskCopyService {
 			lesson: params.destinationLesson,
 		});
 
-		const elements = [...this.defaultTaskStatusElements(), ...this.copyTaskFiles(params.originalTask)];
```

Comment:
Why did you remove this? It's needed for files attached to a task...

Reply:
Rechecked - result: 
Changes are okay as they are, as files need to be attached after the task was persisted... which happens in the task-copy-uc.